### PR TITLE
Fix the problem that dashboard do not correctly show the SystemRule in CPU usage strategy

### LINKED ISSUE #905: SystemRule usage
## Issue Description
rule do not work all and the olny rule which works but it do not set as i want 
Type: *bug report* 

> I set five system rule,use zookeeperdatasource,now receive system rule update data
image

>then i use  curl http://127.0.0.1:8720/getRules?type=system to get my rule

>debug find  [SystemRuleManager ](https://github.com/alibaba/Sentinel/blob/master/sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/system/SystemRuleManager.java)  code like that
image
>only first rule can seem to be seen, someone like the first rule,but the only rule become five rule
image

>also dashboard can not show system which i set highestCpuUsage
image

image

### Tell us your environment
dubbo 2.5.3
sentinel version 1.6.2

### PR DESCRIPTION
**Pull request by Crazy10552**
### Does this pull request fix one issue?
   Fixes #905 

### Describe how you did it
  
-  I add new properties in entity [com.alibaba.csp.sentinel.dashboard.datasource.entity.rule.SystemRuleEntity]

- I update the related js and html about  System rule 

### Describe how to verify it
   
image

### Special notes for reviews
 None

### CODE REVIEW

**Comment by sczyh30** [comment]
Hi, thanks for contributing. Could you please [sign the CLA here](https://cla-assistant.io/alibaba/Sentinel?pullRequest=922)? And please make sure the email of your commits match your GitHub email.

---

感谢贡献，请将 commit 对应的 email 调整成与 GitHub 的 email 相匹配并 [确认一下 CLA](https://cla-assistant.io/alibaba/Sentinel?pullRequest=922)。

File: sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/datasource/entity/rule/SystemRuleEntity.java [hunk 1]
```diff
@@ -33,6 +33,7 @@
     private Long avgRt;
     private Long maxThread;
     private Double qps;
+    private Double avgCpu;
```
Comment (sczyh30) [inline sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/datasource/entity/rule/SystemRuleEntity.java]: Maybe it's better to be `maxCpuUsage`?

File: sentinel-dashboard/src/main/webapp/resources/app/scripts/controllers/system.js [hunk 2]
```diff
@@ -119,6 +121,9 @@ app.controller('SystemCtl', ['$scope', '$stateParams', 'SystemService', 'ngDialo
       } else if (rule.qps != -1) {
         ruleTypeDesc = 'QPS';
         ruleTypeCount = rule.qps;
+      }else if (rule.avgCpu != -1) {
```
Comment (sczyh30) [inline sentinel-dashboard/src/main/webapp/resources/app/scripts/controllers/system.js]: Ditto (`maxCpuUsage`)

File: sentinel-dashboard/src/main/webapp/resources/app/views/system.html [hunk 3]
```diff
@@ -49,12 +49,14 @@
                   <span ng-if="rule.avgRt >= 0">RT</span>
                   <span ng-if="rule.maxThread >= 0">线程数</span>
                   <span ng-if="rule.qps >= 0">QPS</span>
+                  <span ng-if="rule.avgCpu >= 0">CPU</span>
```
Comment (sczyh30) [inline sentinel-dashboard/src/main/webapp/resources/app/views/system.html]: 叫 `CPU 使用率` 可能更合适些？

File: sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/datasource/entity/rule/SystemRuleEntity.java (hunk 1 above)
Comment (Crazy10552) [reply sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/datasource/entity/rule/SystemRuleEntity.java]: I saw properties in [SystemRule ](https://github.com/alibaba/Sentinel/blob/master/sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/system/SystemRule.java) named highestSystemLoad and  it named avgLoad in [SystemRuleEntity](https://github.com/alibaba/Sentinel/blob/master/sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/datasource/entity/rule/SystemRuleEntity.java) ,so i just do like to name highestCpuUsage in [SystemRule ](https://github.com/alibaba/Sentinel/blob/master/sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/system/SystemRule.java) to avgCpu in [SystemRuleEntity](https://github.com/alibaba/Sentinel/blob/master/sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/datasource/entity/rule/SystemRuleEntity.java)

File: sentinel-dashboard/src/main/webapp/resources/app/views/system.html (hunk 3 above)
Comment (Crazy10552) [reply sentinel-dashboard/src/main/webapp/resources/app/views/system.html]: i know it's much better but  i just confused by the AvgLoad ，i can fix both two of them

File: sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/datasource/entity/rule/SystemRuleEntity.java (hunk 1 above)
Comment (sczyh30) [reply sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/datasource/entity/rule/SystemRuleEntity.java]: Other legacy fields might not be good enough... For CPU usage it's an instantaneous value rather that average value.

File: sentinel-dashboard/src/main/webapp/resources/app/views/system.html (hunk 3 above)
Comment (sczyh30) [reply sentinel-dashboard/src/main/webapp/resources/app/views/system.html]: It's indeed confusing. You could fix that.

**Comment by sczyh30** [comment]
另外将 commit 对应的 email 调整成与 GitHub 的 email 相匹配并 [确认一下 CLA](https://cla-assistant.io/alibaba/Sentinel?pullRequest=922) 哈。可以参考：https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user

File: sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/controller/SystemController.java [hunk 4]
```diff
@@ -164,11 +221,23 @@ private int countNotNullAndNotNegative(Number... values) {
         if (StringUtil.isNotBlank(app)) {
             entity.setApp(app.trim());
         }
-        if (avgLoad != null) {
-            if (avgLoad < 0) {
-                return Result.ofFail(-1, "avgLoad must >= 0");
+        if (highestSystemLoad != null) {
```
Comment (sczyh30) [inline sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/controller/SystemController.java]: `highestSystemLoad` 的取值范围是 [0, +∞)，load 没有 < 1 的限制。这个需要保持与之前一致。

File: sentinel-dashboard/src/main/webapp/resources/app/views/system.html (hunk 3 above)
Comment (Crazy10552) [reply sentinel-dashboard/src/main/webapp/resources/app/views/system.html]: image
I left a problem that i am not good at,while i try to choose another ,it does not show like i want 
image

File: sentinel-dashboard/src/main/webapp/resources/app/views/dialog/system-rule-dialog.html [hunk 5]
```diff
@@ -9,40 +9,41 @@
...
+                <input type="radio" name="grade" value="3" ng-model='currentRule.grade' ng-disabled="systemRuleDialog.type == 'edit'"/>&nbsp;入口 QPS
+                <!--highestCpuUsage -->
+                <input type="radio" name="grade" value="4" checked ng-model='currentRule.grade' ng-disabled="systemRuleDialog.type == 'edit'" />&nbsp;最大CPU使用&nbsp;&nbsp;
+
               </div>
               <div class="form-control highlight-border" ng-if="systemRuleDialog.type == 'add'" align="center">
-                <!--avgLoad -->
-                <input type="radio" name="grade" value="0" ng-model='currentRule.grade' ng-disabled="systemRuleDialog.type == 'edit'" />&nbsp;LOAD&nbsp;&nbsp;
+                <!--highestSystemLoad -->
+                <input type="radio" name="grade" value="0" ng-model='currentRule.grade' ng-disabled="systemRuleDialog.type == 'edit'" />&nbsp;最大系统LOAD&nbsp;&nbsp;
```
Comment (sczyh30) [inline sentinel-dashboard/src/main/webapp/resources/app/views/dialog/system-rule-dialog.html]: 这里是阈值，不需要强调”最大“，这里直接叫 Load 即可

File: sentinel-dashboard/src/main/webapp/resources/app/views/dialog/system-rule-dialog.html [hunk 6]
```diff
@@ -9,40 +9,41 @@
...
             <div class="col-sm-9">
-              <input type='number' class="form-control highlight-border" ng-model='currentRule.avgLoad' placeholder="[0, ~)的小数" ng-if="currentRule.grade == 0"
-              />
-              <input type='number' class="form-control highlight-border" ng-model='currentRule.avgRt' placeholder="[0, ~)的整数" ng-if="currentRule.grade == 1"
-              />
-              <input type='number' class="form-control highlight-border" ng-model='currentRule.maxThread' placeholder="[0, ~)的整数" ng-if="currentRule.grade == 2"
-              />
-              <input type='number' class="form-control highlight-border" ng-model='currentRule.qps' placeholder="[0, ~)的小数" ng-if="currentRule.grade == 3"
-              />
+              <input type='number' class="form-control highlight-border" ng-model='currentRule.highestSystemLoad' placeholder="[0, 1)的小数" ng-if="currentRule.grade == 0"/>
```
Comment (sczyh30) [inline sentinel-dashboard/src/main/webapp/resources/app/views/dialog/system-rule-dialog.html]: load 的取值保持与之前一致

File: sentinel-dashboard/src/main/webapp/resources/app/views/system.html [hunk 7]
```diff
@@ -45,16 +45,18 @@
               <tr dir-paginate="rule in rules | filter : searchKey | itemsPerPage: rulesPageConfig.pageSize " current-page="rulesPageConfig.currentPageIndex"
                 pagination-id="entriesPagination">
                 <td style="word-wrap:break-word;word-break:break-all;">
-                  <span ng-if="rule.avgLoad >= 0">LOAD</span>
+                  <span ng-if="rule.highestSystemLoad >= 0">最大系统LOAD</span>
```
Comment (sczyh30) [inline sentinel-dashboard/src/main/webapp/resources/app/views/system.html]: ditto

File: sentinel-dashboard/src/main/webapp/resources/app/views/system.html [hunk 8]
```diff
@@ -45,16 +45,18 @@
...
                   <span ng-if="rule.avgRt >= 0">RT</span>
                   <span ng-if="rule.maxThread >= 0">线程数</span>
                   <span ng-if="rule.qps >= 0">QPS</span>
+                  <span ng-if="rule.highestCpuUsage >= 0">最大CPU使用率</span>
```
Comment (sczyh30) [inline sentinel-dashboard/src/main/webapp/resources/app/views/system.html]: ditto

File: sentinel-dashboard/src/main/webapp/resources/app/views/dialog/system-rule-dialog.html [hunk 9]
```diff
@@ -9,40 +9,41 @@
...
                 <!--avgRt -->
                 <input type="radio" name="grade" value="1" ng-model='currentRule.grade' ng-disabled="systemRuleDialog.type == 'edit'" />&nbsp;RT&nbsp;&nbsp;
                 <!--maxThread -->
                 <input type="radio" name="grade" value="2" ng-model='currentRule.grade' ng-disabled="systemRuleDialog.type == 'edit'" />&nbsp;线程数&nbsp;&nbsp;
                 <!--qps -->
-                <input type="radio" name="grade" value="3" checked ng-model='currentRule.grade' ng-disabled="systemRuleDialog.type == 'edit'"
-                />&nbsp;入口 QPS
+                <input type="radio" name="grade" value="3" ng-model='currentRule.grade' ng-disabled="systemRuleDialog.type == 'edit'"/>&nbsp;入口 QPS
+                <!--highestCpuUsage -->
+                <input type="radio" name="grade" value="4" checked ng-model='currentRule.grade' ng-disabled="systemRuleDialog.type == 'edit'" />&nbsp;最大CPU使用&nbsp;&nbsp;
```
Comment (sczyh30) [inline sentinel-dashboard/src/main/webapp/resources/app/views/dialog/system-rule-dialog.html]: CPU 使用率

File: sentinel-dashboard/src/main/webapp/resources/app/views/dialog/system-rule-dialog.html (hunk 5 above)
Comment (Crazy10552) [reply sentinel-dashboard/src/main/webapp/resources/app/views/dialog/system-rule-dialog.html]: i see

**Comment by sczyh30** [comment]
另外需要将 commit 对应的 email 调整成与 GitHub 的 email 相匹配并 [确认一下 CLA](https://cla-assistant.io/alibaba/Sentinel?pullRequest=922) 。可以参考：https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user

**Pull request closed without merging**